README.md

```diff
@@ -66,6 +66,9 @@ to instantiate WavefrontProxyClient or WavefrontDirectIngestionClient.
 **Note:** If you are using more than one Wavefront SDK (i.e. wavefront-opentracing-sdk-java, wavefront-dropwizard-metrics-sdk-java, wavefront-jersey-sdk-java, wavefront-grpc-sdk-java etc.) that requires you to instantiate WavefrontSender, then you should instantiate the WavefrontSender only once and share that sender instance across multiple SDKs inside the same JVM.
 If the SDKs will be installed on different JVMs, then you would need to instantiate one WavefrontSender per JVM.
 
+### WavefrontTracer
+To enable sending tracing spans from the SDK, we need to instantiate a WavefrontTracer. Refer to this page (https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java#tracer) for more details. 
+
```

Comment:
We should also mention how to use the withApplicationTags() method for the tracer.
That is not mentioned on https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java#tracer

I strongly suggest add that code snippet here along with reference to https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java#tracer